platform/security/common/src/main/java/org/codice/ddf/security/common/Security.java

```diff
@@ -69,18 +72,33 @@
  */
 public class Security {
 
-  private static final Security INSTANCE = new Security();
-
   private static final Logger LOGGER = LoggerFactory.getLogger(Security.class);
 
+  private static final Security INSTANCE = new Security();
+
   private static final String INSUFFICIENT_PERMISSIONS_ERROR =
       "Current user doesn't have sufficient privileges to run this command";
 
   private static final String KARAF_LOCAL_ROLE = "karaf.local.roles";
 
+  private static final javax.security.auth.Subject JAVA_ADMIN_SUBJECT = getAdminJavaSubject();
+
   private Subject cachedSystemSubject;
 
-  private static final javax.security.auth.Subject JAVA_ADMIN_SUBJECT = getAdminJavaSubject();
+  protected static final int MAX_RETRIES = 10;
+
+  // using a Supplier here to prevent threading issues
+  private Supplier<RetryPolicy> retryPolicySupplier =
```

Comment:
❓ Should we consider making this package-private instead of the constructor? This would allow the unit tests to replace it directly instead of using a constructor, and would also allow us to add a test to make sure that the default `RetryPolicy` is properly configured.

Reply:
https://github.com/codice/ddf/blob/ec941adff4bd53110b902fb6fb55248ff2ce9691/platform/security/common/src/test/java/org/codice/ddf/security/common/SecurityTest.java#L204-L224